<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLSegmentationDisplayNodeWidget</class>
 <widget class="qMRMLWidget" name="qMRMLSegmentationDisplayNodeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qMRMLSegmentationDisplayNodeWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="3">
    <widget class="ctkSliderWidget" name="SliderWidget_Opacity">
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="pageStep">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_OverallOpacity">
     <property name="text">
      <string>Overall opacity:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_Advanced">
     <property name="title">
      <string>Advanced</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="1">
       <widget class="qMRMLDisplayNodeViewComboBox" name="DisplayNodeViewComboBox"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_Views">
        <property name="text">
         <string>Views:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_SliceIntersectionThickness">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_Representation2D">
        <property name="text">
         <string>Representation in 2D views:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_DisplayedRepresentation3D">
        <property name="toolTip">
         <string>Representation that is shown in 3D (models only)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_Representation3D">
        <property name="toolTip">
         <string>Representation that is shown as a model in 3D and as slice intersections in 2D if exists</string>
        </property>
        <property name="text">
         <string>Representation in 3D views:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox_DisplayedRepresentation2D">
        <property name="toolTip">
         <string>Representation that is shown in the 2D slice views</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_SliceIntersectionThickness">
        <property name="text">
         <string>Slice intersection thickness:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_SelectedSegment">
        <property name="title">
         <string>Selected segment</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <property name="spacing">
          <number>4</number>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="label_SliceOutline_SelectedSegment">
           <property name="text">
            <string>Slice outline:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="checkBox_VisibilitySliceFill_SelectedSegment">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_SliceFill_SelectedSegment">
           <property name="text">
            <string>Slice fill:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="checkBox_VisibilitySliceOutline_SelectedSegment">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLabel" name="label_Visibility_SelectedSegment">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Visibility</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>8</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="checkBox_Visibility3D_SelectedSegment">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3D_SelectedSegment">
           <property name="text">
            <string>3D:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="ctkSliderWidget" name="SliderWidget_OpacitySliceFill_SelectedSegment">
           <property name="toolTip">
            <string>Value relative to other segments. The final opacity depends both on the per-segment opacity and the overall opacity (above)</string>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
           <property name="pageStep">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="ctkSliderWidget" name="SliderWidget_OpacitySliceOutline_SelectedSegment">
           <property name="toolTip">
            <string>Value relative to other segments. The final opacity depends both on the per-segment opacity and the overall opacity (above)</string>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
           <property name="pageStep">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="ctkSliderWidget" name="SliderWidget_Opacity3D_SelectedSegment">
           <property name="toolTip">
            <string>Value relative to other segments. The final opacity depends both on the per-segment opacity and the overall opacity (above)</string>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
           <property name="pageStep">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_Opacity_SelectedSegment">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Opacity</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="checkBox_Visible">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QFrame" name="frame_VisibilityOpacity">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="label_SliceFill">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Slice fill:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_SliceOutline">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Slice outline:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3D">
        <property name="text">
         <string>3D:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_Opacity">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Opacity</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="ctkSliderWidget" name="SliderWidget_OpacitySliceOutline">
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="pageStep">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="ctkSliderWidget" name="SliderWidget_OpacitySliceFill">
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="pageStep">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="ctkSliderWidget" name="SliderWidget_Opacity3D">
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="pageStep">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>8</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="checkBox_VisibilitySliceFill">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLabel" name="label_Visibility">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Visibility</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="checkBox_VisibilitySliceOutline">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="checkBox_Visibility3D">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_Visible">
     <property name="text">
      <string>Overall visibility:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox">
     <property name="title">
      <string>Clipping</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="3" column="0" colspan="3">
       <widget class="qMRMLClipNodeWidget" name="SlicerWidget_ClipNodeProperties"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_ClipNode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Clip node:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_Clip">
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLClipNode</string>
         </stringlist>
        </property>
        <property name="showHidden">
         <bool>true</bool>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="qMRMLClipNodeDisplayWidget" name="SlicerWidget_ClipNodeDisplayProperties"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLCheckableNodeComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLCheckableNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLClipNodeDisplayWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLClipNodeDisplayWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLClipNodeWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLClipNodeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLDisplayNodeViewComboBox</class>
   <extends>qMRMLCheckableNodeComboBox</extends>
   <header>qMRMLDisplayNodeViewComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qMRMLSegmentationDisplayNodeWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_Clip</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qMRMLSegmentationDisplayNodeWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>SlicerWidget_ClipNodeDisplayProperties</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qMRMLSegmentationDisplayNodeWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>SlicerWidget_ClipNodeProperties</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
